docs/src/components/pricing/FAQ.tsx

```diff
@@ -63,6 +58,34 @@ const faqData = [
       </React.Fragment>
     ),
   },
+  {
+    summary: 'Why do we must license developers not using the software directly?',
+    detail: (
+      <React.Fragment>
+        Our pricing model requires all developers working on a project using MUI X Pro or Premium to
+        be licensed. This is designed to make it easier for your engineers to use the software
+        without having to constantly verify if they have the right number of seats. It aims to
+        replicate one of the great properties open-source licenses have: simplicity.
+        <br />
+        <br />
+        Our pricing model also requires developers indirectly using MUI X Pro or Premium to be
+        licensed, e.g. through a wrapper library. This is because the more developers in your
+        organization using the software, the more they need comprehensive documentation, and the
```

Comment:
```suggestion
```

Reply:
I have shortened this, the aim to to push a bit our agenda on: no building in-house won't be a 2 months task.